autocompleter/widgets.py

```diff
@@ -30,3 +34,84 @@ def build_attrs(self, *args, **kwargs):
             'data-autocompleter-url': self.autocompleter_url
         })
         return attrs
+
+
+class AutocompleterSelectWidget(forms.MultiWidget):
+    """
+    This widget renders two adjacent <input> elements that provide a clean user interface
+    for searching via the Autocompleter API & selecting the desired object.
+
+    Wrapper for 2 widgets:
+        1. `AutocompleterWidget`: renders an <input> that acts as the search field.
+        2. `HiddenInput`: renders an <input> that holds the actual DB value (which is some kind of GUI)
+
+
+    :autocompleter_name - `str` specify the Autocompleter to use for search.
+    :display_name_field - `str` specify field from payload to display to user when result is selected.
+    :database_field     - `str` specify field from payload to save to the DB (should be a GUI).
+    :object_resolver    - `callable` that can fetch the object using the `database_field` value.
+    """
+
+    def __init__(self, autocompleter_name, autocompleter_url, display_name_field,
```

Comment:
can we not get the `autocompleter_url` from the `autocompleter_name`? Seems weird to have to pass both in.